.github/workflows/bot.yml

```diff
@@ -545,22 +511,6 @@ jobs:
             flinkProfile: 'flink1.16'
```

Comment:
Make sure you track a JIRA that adds Flink 1.15 and 1.14 bundle validation in the version matrix.

Reply:
https://issues.apache.org/jira/browse/HUDI-8002